Sources/TuistCore/Graph/Mappers/TargetMapper.swift

```diff
@@ -3,3 +3,22 @@ import Foundation
 public protocol TargetMapping {
     func map(target: Target) throws -> (Target, [SideEffectDescriptor])
 }
+
+public class TargetActionEnvironmentMapper: TargetMapping {
```

Comment:
minor: The implementation can move too https://github.com/tuist/tuist/pull/1782#discussion_r493557151

 but happy to do this afterwards too - you've suffered my feedback long enough :p

Reply:
I have nothing but appreciation for your help. I'll move this over, too.

I have a personal aversion to "Can I fix this in a follow-up PR?" because in my experience, the follow-up rarely materializes.